### ISSUE TITLE
after using setbg new instances of st doesnt remain their bg color

### ISSUE THREAD

**Issue by void00r** [LukeSmithxyz/voidrice#767]
![pic-full-200719-1501-42](https://user-images.githubusercontent.com/62638787/87871763-eaecf600-c9d0-11ea-9e8c-e0eaf86b43a3.png)

**Comment by narukeh** [comment]
if you have pywal installed, that's what it is supposed to do, change the color scheme.
(pywal sometimes fails to change the colorscheme of already present windows)
just update `st` to eliminate variables:
```
cd /tmp
git clone 'https://github.com/LukeSmithxyz/st' && cd st
sudo make install
```

**Comment by void00r** [comment]
yeah i did update st with lukes new latest one. And as a matter of fact it seems like fixed the problem but there is a still little bit of polishing left here. when i run bg command on any wallpaper through vifm or lf that partitcular windows transparency and color changes and but new st instances arent seems dark as those and it feel like yellowish background color in every new st's.

**Comment by narukeh** [comment]
Like i said pywal cant change 100% the`already opened window`'s color scheme. The new windows have the color that they should have. 
Here is what happens when i `setbg` the image that already is the `bg`.

![pic-full-200720-0947-46](https://user-images.githubusercontent.com/62762955/87912707-22f64680-ca6e-11ea-8e6f-381328b273a9.png)


(Tho they shouldn't be yellowish). Try a wallpaper that doesn't have yellow 😀.  Pywal sometimes picks minority colors as a main one. (if nothig, than ->1)
1. Remove `~/.config/Xresources` and see what happens 🤷
(if nothig, than ->2)
2. `xrdb ~/.cache/wal/colors.Xresources`

**Comment by void00r** [comment]
I dont have this flle in my system colors.Xresources

**Comment by narukeh** [comment]
@void00r  You do. Unless you modified "`~/.local/bin/setbg`". Just run setbg again, and "`~/.cache/wal/colors.Xresources`" will be generated (along whith other stuff).

**Comment by void00r** [comment]
Now that I changed the bg color of st from #282828 to #000000 (black). It does not have yellowish color but blackish. and It looks like i had colors.xresources instead of colors.Xresources. but i renamed it and run set bg script and new file with small x created colors.xresources. and yeah the bg is blackish as i changed it st build.

**Comment by void00r** [comment]
And I am pretty sure i didnt made any changes in "~/.local/bin/setbg"

**Comment by narukeh** [comment]
if the colorscheme in `st`'s `config.h` caused the problems, then i recommend to change ALL to the default: http://www.calmar.ws/vim/256-xterm-24bit-rgb-color-chart.html

> And I am pretty sure i didnt made any changes in "~/.local/bin/setbg"

if your not sure, just grab the [latest one](https://raw.githubusercontent.com/LukeSmithxyz/voidrice/master/.local/bin/setbg), better yet update larbs.

**Comment by void00r** [comment]
Alright I grabbed the latest one. And everythings sets right now. At first run of setbg it changes bg color of every st terminal presents at the time. but new instances still get the resets bg color from st's config.h. for my convinience I am satisfied as I changed the bg color to complete black. It looks nice to me. 
but these issue still remain:

1. New instances of st wont get color from xresources or pywal or setbg.
2. For some reason my "~/.cache/wal" have "colors.xresources" instead of "colors.Xresources"
